src/components/app-controllers/SingleAppController/SingleAppController.js

```diff
@@ -36,52 +40,52 @@ export default createReactClass({
   datasetFilter: function (datafile) {
-    return !(datafile.multi_year_mean == false && datafile.timescale == "monthly");
+    return !(datafile.multi_year_mean === false && datafile.timescale === 'monthly');
   },
 
   //Returns metadata for datasets with thethe selected variable + scenario, any model.
   //Passed as a prop for SingleDataController to generate model comparison graphs.
   getModelContextMetadata: function () {
     return _.where(this.state.meta,
-        {variable_id: this.state.variable_id, experiment: this.state.experiment});
+        { variable_id: this.state.variable_id, experiment: this.state.experiment });
   },
 
   render: function () {
     //hierarchical selection: model, then variable, then experiment
     var modOptions = this.getMetadataItems('model_id');
     var expOptions = this.markDisabledMetadataItems(this.getMetadataItems('experiment'),
-        this.getFilteredMetadataItems('experiment', {model_id: this.state.model_id, variable_id: this.state.variable_id}));
+        this.getFilteredMetadataItems('experiment', { model_id: this.state.model_id, variable_id: this.state.variable_id }));
 
     // TODO: https://github.com/pacificclimate/climate-explorer-frontend/issues/122
     // TODO: https://github.com/pacificclimate/climate-explorer-frontend/issues/125
     return (
       <Grid fluid>
         <Row>
           <Col lg={4} md={4}>
-            <Selector 
-              label={"Model Selection"} 
-              onChange={this.updateSelection.bind(this, 'model_id')} 
-              items={modOptions} 
+            <Selector
+              label={modelSelectorLabel}
+              onChange={this.updateSelection.bind(this, 'model_id')}
+              items={modOptions}
               value={this.state.model_id}
             />
           </Col>
-          <Col lg={4} md={4}>
-            <VariableDescriptionSelector
-              label={"Variable Selection"}
-              onChange={this.handleSetVariable.bind(this, "variable")}
-              meta={this.state.meta}
-              constraints={{model_id: this.state.model_id}}
-              value={_.pick(this.state, "variable_id", "variable_name")} 
-            />
-          </Col>
           <Col lg={4} md={4}>
             <Selector
-              label={"Emission Scenario Selection"}
+              label={emissionScenarioSelectorLabel}
               onChange={this.updateSelection.bind(this, 'experiment')}
               items={expOptions}
               value={this.state.experiment}
             />
           </Col>
+          <Col lg={4} md={4}>
+            <VariableDescriptionSelector
+              label={variableSelectorLabel}
+              onChange={this.handleSetVariable.bind(this, 'variable')}
+              meta={this.state.meta}
+              constraints={{ model_id: this.state.model_id }}
+              value={_.pick(this.state, 'variable_id', 'variable_name')}
+            />
+          </Col>
         </Row>
```

Comment:
If you change the order of the selectors (swapping Variable and Emissions Scenario) you may want to change the "hierarchy" of the selectors as well to make it more intuitive.

Previously, the selectors were arranged model -> variable -> emissions from left to right. Model allowed the user to select any models. Variable allowed the user to select only variables available for the selected model. Emissions allows the user to select only emissions available for the selected model and variable. There was an implicit left to right hierarchy to the selectors such that each selector would disable all items that didn't make sense based on the "already made" choices in selectors to the left.

Now going from left to right we have model -> emissions -> variable, but we've left the item-availability hierarchy model -> variable -> emissions.

Reply:
... though switching the hierarchy to model -> emissions -> variable gives you a different UI awkwardness: the user has to guess the "right" emissions scenario before they can select the variable they want, and if they select the "wrong" emissions scenario, the variable they want will be disabled. 

That setup was unfortunately necessary for the dual-variable portal, since the two variable are required to share a model and emissions scenario. I assume you switched the order to make the portals more consistent, which makes sense.

What the user really cares about is probably mostly the variable. If we had only this portal, it might make most sense to set up the selectors and hierarchy variable -> emissions scenario -> model, but there's some value in consistency across portals.